### ISSUE TITLE
Develop the whole site

### ISSUE THREAD

**Issue by ng-david** [gotechnica/technica-site-2017#41]
(And registration)

By end of JULY

**Comment by eveninglily** [comment]
Would be awesome if we could also get https setup for the site this year, https://letsencrypt.org/ is free and widely used, not too hard to setup, no matter where the site is hosted. I recently set it up for umd.io, which runs on digitalocean

**Comment by ng-david** [comment]
Good call! Please make a ticket on ZenHub and assign it to me (and you if
you'd like :D)
